umcontroller/umcontroller.go

```diff
@@ -1135,7 +1140,7 @@ func (umCtrl *Controller) processUpdateUpdateStatus(ctx context.Context, e *fsm.
 		}
 	}
 
-	umCtrl.updateCurrentComponentsStatus(status.componsStatus)
+	umCtrl.updateCurrentComponentsStatus(status.componsStatus, umID)
```

Comment:
Using `umID` looks confusing now. We should rename it to `nodeID` in all the cases. Could you do it in separate commit this PR?